tests/test_room.py

```diff
@@ -17,33 +23,119 @@ def test_create_room(self):
         daedalus = Office("Daedalus")
         abydos = LivingSpace("Abydos")
         self.assertEqual(daedalus.name, "Daedalus")
-        self.assertEqual(daedalus.type, "Office")
-        self.assertEqual(daedalus.max_persons, 6)
+        self.assertEqual(daedalus.room_type, "Office")
+        self.assertEqual(daedalus.capacity, 6)
         self.assertEqual(abydos.name, "Abydos")
-        self.assertEqual(abydos.type, "Living Space")
-        self.assertEqual(abydos.max_persons, 4)
+        self.assertEqual(abydos.room_type, "Living Space")
+        self.assertEqual(abydos.capacity, 4)
+        del(daedalus)
+        del(abydos)
```

Comment:
Having a teardown method would be preferable as compared to deleting the values. Imagine if there are 1000 records. Teardown also ensures that your tests are isolated. Currently your tests are not isolated and if the order in which your tests are run changes some may fail.